src/components/Avatar/index.js

```diff
@@ -37,6 +37,8 @@ const Avatar = ({
     };
   }, [size]);
 
+  const [loadedImageUrl, setLoadedImageUrl] = useState(null);
```

Comment:
We don't care about the actual url, do we?

What do you think of renaming this to `loaded` and making it a boolean?

Reply:
agreed! updated